code/dynamic-proxy/server.js

```diff
@@ -0,0 +1,21 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+import etcdRoutes from './api/etcd.routes';
+import startProxy from './proxy/proxy.js';
+import config from './config';
+import bunyan from 'bunyan';
+
+const logger = bunyan.createLogger({name: 'proxy.api'});
```

Comment:
Minor liniting issue, add spaces for object curly braces. `{ name: 'proxy.api' }`